apps/extension/src/scanner/codeScanner.ts

```
@@ -0,0 +1,83 @@
+        }
+
+        try {
+            logger.info('Scanning', document.fileName);
+            const result = await scan({
+                sessionId,
+                filePath: document.uri.fsPath,
+                languageId: document.languageId,
+                content,
+            });
+            logger.info(`Scan complete — ${result.vulnerabilities.length} issue(s)`, document.fileName);
+            this.listeners.forEach(l => l(document.uri, result));
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)